core/chaincode/lifecycle/endorsment_info.go

```diff
@@ -90,6 +92,12 @@ func (cei *ChaincodeEndorsementInfoSource) CachedChaincodeInfo(channelID, chainc
 	}
 
 	if chaincodeInfo.InstallInfo == nil {
+		if cei.UserRunsCC {
+			chaincodeInfo.InstallInfo = &ChaincodeInstallInfo{
+				PackageID: fmt.Sprintf("%s:%s", chaincodeName, chaincodeInfo.Definition.EndorsementInfo.Version),
```

Comment:
In a follow-on, please consider replacing the printf with concatenation.

Reply:
Done